packages/jellyfish-transaction/src/constant/Regex.ts

```diff
@@ -0,0 +1,2 @@
+export const BASE_58 = new RegExp(/[1-9A-HJ-NP-Za-km-z]/)
+export const BECH_32 = new RegExp(/[0-9ac-hj-np-z]/)
```

Comment:
I personally hate the use of constants, it requires unnecessary code cross-referencing when the constants are never used beyond the scope of where it's declared. And for constants that never change, it's kinda pointless to have an extra file for it.